test/e2e/policies.test-suite/podpools/n4c16/test06-prometheus-metrics/code.var.sh

```diff
@@ -97,6 +97,8 @@ CPUREQ="" CPULIM=""
 namespace=kube-system CONTCOUNT=3 create podpools-busybox
 report allowed
 vm-command "curl --silent $metrics_url | grep -v ^cgroup_"
-verify-log-vs-metrics pod5:pod5c1 0 20
+# There should be kube-apiserver, etcd etc. running on reserved CPUs as well,
+# therefore allow a lot of CPU usage yet pod5 is not doing anything.
+verify-log-vs-metrics pod5:pod5c1 0 100
```

Comment:
Is this somehow related?

Reply:
Yes, strongly. Now that the fix in policy Start() makes sure that kube-apiserver and etcd (among others) will be pinned to the reserved CPU - the same CPU where pod5 is scheduled too - the CPU load of that CPU can be much higher than at the time when this test was written. I got one failed test run because of that: the two processes may easily cause > 20% load on the only reserved CPU in our e2e VMs.

Reply:
Thanks 👍